task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,13 @@
  *
  */
 function getFizzBuzz(num) {
```

Comment:
fix according to google's style guide

https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks